packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/CozyMarkerBuilder.java

```diff
@@ -40,108 +40,143 @@ private static Paint setTextPaint(float size, Context context) {
         return paint;
     }
 
-    @NonNull
-    private static Paint getBackgroundColor() {
+    private Paint getMarkerPaint() {
         Paint paint = new Paint();
         paint.setColor(Color.WHITE);
         paint.setAntiAlias(true);
         return paint;
     }
 
-    @NonNull
-    private static Paint getShadowPaint() {
+    private Paint getShadowPaint() {
         Paint paint = new Paint();
         paint.setColor(Color.BLACK);
         paint.setStyle(Paint.Style.STROKE);
         paint.setAlpha(15);
-        paint.setStrokeWidth(6);
-        paint.setMaskFilter(new BlurMaskFilter(8, BlurMaskFilter.Blur.NORMAL));
+        paint.setStrokeWidth(shadowSize);
         paint.setAntiAlias(true);
         return paint;
     }
 
-    private static int getMarkerSize() {
+    private int getMarkerSize() {
         int baseScreenHeight = 2467;
         int baseMarkerSize = 167;
         int maxMarkerSize = 172;
         int minMarkerSize = 67;
 
         int physicalPixelHeight = Resources.getSystem().getDisplayMetrics().heightPixels;
-        double heightRatio = ((double ) (physicalPixelHeight)) / ((double) (baseScreenHeight));
+        double heightRatio = ((double) (physicalPixelHeight)) / ((double) (baseScreenHeight));
         int proportionalMarkerSize = (int) (baseMarkerSize * heightRatio);
 
         if (proportionalMarkerSize > maxMarkerSize) {
             return maxMarkerSize;
-        } else if (proportionalMarkerSize < minMarkerSize) {
-            return minMarkerSize;
-        } else {
-            return proportionalMarkerSize;
-        }
+        } else
+            return Math.max(proportionalMarkerSize, minMarkerSize);
     }
 
-    private static Bitmap getClusterBitmap(int size) {
+    private int getBorderRadius(int multiply) {
+        double density = Resources.getSystem().getDisplayMetrics().density;
+        return (int) (density * multiply);
+    }
```

Comment:
Can you elaborate on what this is supposed to be? (I don't code java anymore 🤷 )

Reply:
It was a method used to get the border radius based on the screen's [density](https://developer.android.com/reference/android/util/DisplayMetrics#density)... but this method in a bigger device returns a slightly different value which didn't really followed the marker's design, it made it more "squarey"...
I thought about removing this method and setting a fixed value so the border radius would be the same regardless of the device on a recent commit... would we be ok with that?